html/index.html

```diff
@@ -71,8 +77,67 @@ <h2 class="landing__title">LAND YOUR SERVICES IN STYLE</h2>
     </div>
 
     <div class="landing">
-        <h2 class="landing__title">WHAT OUR CLIENTS SAYS</h2>
+        <h2 class="title__white">WHAT OUR CLIENTS SAYS</h2>
         <h3 class="landing__paragraph">"Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla pretium lectus vel Lorem ipsum dolor sit amet, consectetur adipiscing elit."</h3>
+        <span class="landing__paragraph flex-self-end"><strong>-Jimmy Kelly</strong></span>
     </div>
+
+    <div class="developers">
+        <div class="title__black">Our developers</div>
+        <div class="row-fluid developers__container">
+            <div class="developers__section">
+                <div class="developers__section-image"></div>
+                <h4 class="developers__section-name">Cheiu Linewasa</h4>
+                <p class="developers__section-position">Lead Designer</p>
+                <p class="developers__section-bio">Aenean faucibus luctus enim. Duis quis sem risu suspend lacinia elementum nunc.</p>
+            </div>
+            <div class="developers__section">
+                <div class="developers__section-image"></div>
+                <h4 class="developers__section-name">Aexano Teranxsa</h4>
+                <p class="developers__section-position">Serior Analyst</p>
+                <p class="developers__section-bio">Aenean faucibus luctus enim. Duis quis sem risu suspend lacinia elementum nunc.</p>
+            </div>
+            <div class="developers__section">
+                <div class="developers__section-image"></div>
+                <h4 class="developers__section-name">Zineesa Renaiqn</h4>
+                <p class="developers__section-position">Lead Developer</p>
+                <p class="developers__section-bio">Aenean faucibus luctus enim. Duis quis sem risu suspend lacinia elementum nunc.</p>
+            </div>
+        </div>
+    </div>
+
+    <div class="license">
+        <div class="license__single">
+            <span class="license__price">199 $</span>
+            <span class="license__note">single license</span>
+        </div>
+        <div class="license__multiple">
+            <span class="license__price">890 $</span>
+            <span class="license__note">multiple license</span>
+        </div>
+    </div>
+
+    <footer class="footer">
+        <div class="footer__connected">
+            <div class="title__black">Stay connected</div>
+            <div class="footer__about-section row-fluid">
```

Comment:
Try to think if this section could be used somewhere else on the site? If yes, it could be standalone block instead of footer element.

Reply:
Not fully got what you mean. I like to put footer element mostly for intuition, when somebody else is reading it.